src/recoil_values/Recoil_atomFamily.js

```diff
@@ -235,10 +235,7 @@ function atomFamily<T, P: Parameter>(
     const newAtom = atom<T>({
       key: `${options.key}__${stableStringify(params) ?? 'void'}`,
       default: atomFamilyDefault(params),
-      scopeRules_APPEND_ONLY_READ_THE_DOCS: mapScopeRules(
-        options.scopeRules_APPEND_ONLY_READ_THE_DOCS,
-        params,
-      ),
+      // @fb-only: scopeRules_APPEND_ONLY_READ_THE_DOCS: mapScopeRules(options.scopeRules_APPEND_ONLY_READ_THE_DOCS, params),
```

Comment:
sorry.. i don't think this would work either 😔 
`// @fb-only` comments are actually generating during the sync build and aren't stored internally
@mondaychen @drarmstr do you know what the process is usually for these when coming from external contributions?

Reply:
actually it seems like the system is smart enough to do the same sync in revers order :) 
can you fix the formatting so it doesn't change the lines?
![Screen Shot 2020-06-12 at 5 45 10 PM](https://user-images.githubusercontent.com/940133/84551874-91f4b880-acd4-11ea-9293-55720c0b586f.png)

Reply:
oh... it even fixed the formatting internally after rebasing. i think this should be good then. @Komalov update the diff and i'll re-import it again